plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -110,6 +110,7 @@ public abstract class BaseJdbcClient
     protected final RemoteQueryModifier queryModifier;
```

Comment:
please move this identifier mapping to plugin-toolkit

Reply:
moved